ui/app/templates/components/clients/history.hbs

```diff
@@ -63,7 +63,7 @@
     {{#if (or @model.startTimeFromLicense this.startTimeFromResponse)}}
```

Comment:
Making this suggestion to fix filter bar showing when no data, config off, and queries available after light testing locally. But there's probably a cleaner way - @Monkeychip may have more insight as I'm not sure why we initially have this check for `(or @model.startTimeFromLicense this.startTimeFromResponse)` instead of something like `(or this.totalUsageCounts this.hasAttributionData)`

```suggestion
    {{#if (or @model.startTimeFromLicense this.startTimeFromResponse)}}
        {{#if @model.endTimeFromResponse}}
```

Reply:
This can be changed to something cleaner as Claire mentioned. Originally all we had was the startTime and endTimes but the conditional she suggested should work too.